docs/content/concepts/reporting/using_stream_report.md

```diff
@@ -0,0 +1,47 @@
+---
+order: 3.5
+type: manual
+entity: reporting
+title: Using Stream Report
+---
+
+### Using Stream Report
+
+The `customer.streamReport()` accepts very similar arguments to `customer.report()`. `page_size` is no longer an optional argument. This is because Googles new streaming method uses chunks rather than pages. Chunk sizes are currently defaulted to 10,000 rows. Hopefully this will be adjustable soon.
+
+This method currently doesn't cache results
+
+The main advantage of `streamReport` is performance. When getting results with a size greater than 1 chunk (10,000 rows), it will fetch those results quicker.
+
```

Comment:
Here is a clearer version of these paragraphs, imo:

The `customer.streamReport()` method is equivalent to `customer.report()`, except that it returns an async generator that you can use to access rows of data as they come back from the API.

Unlike `report`, `streamReport()` does not support the `page_size` argument.

The main advantage of `streamReport` is performance. Not only are you able to get rows before the entire set has finished downloading, but the total time to finish the request will be shorter, especially for large requests (10,000+ rows).

Unfortunately, Google has  hardwired the stream's chunk size to 10,000 rows, so there is no benefit to streaming for smaller requests. We're working with them to find a solution.

Reply:
Have used this copy